Petits-debuts-ui/src/Components/Search/Search.jsx

```diff
@@ -43,7 +48,14 @@ export default function Search({ selectedCategory, setSelectedCategory }) {
     return (
       <div className="search">
         <div className="row-search">
-          <input className="searchBar" type="text" placeholder="type here" />
+          <input
+            className="searchBar"
+            type="text"
+            placeholder="type here"
+            value={search}
+            onChange={(e) => setSearch(e.target.value)}
+          />
+
```

Comment:
can get rid of this space to be consistent

Reply:
Thank you